### PR TITLE
main/spamassassin: Reload spamd after sa-update in crontab

### PR DESCRIPTION
**Pull request by bratkartoffel**
This fixes Bug #10208
https://bugs.alpinelinux.org/issues/10208

### CODE REVIEW

File: main/spamassassin/spamd.crond [hunk 1]
```diff
@@ -1,3 +1,13 @@
 #!/bin/sh
 
-/usr/bin/sa-update
+set -e
+
+. /etc/conf.d/spamd
+pidfile=${pidfile:-/run/spamd.pid}
```
Comment (tcely) [inline main/spamassassin/spamd.crond]:
```suggestion
pidfile="${pidfile:-/run/spamd.pid}"

if [ -f "$pidfile" ]; then
	_pid="$(cat "$pidfile")"
fi
/usr/bin/sa-update
if [ -n "$_pid" ]; then
	kill -SIGHUP "$_pid"
fi
```

File: main/spamassassin/spamd.crond [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+
+if [ -f "$pidfile" ]; then
+	_pid=`cat $pidfile`
+	/usr/bin/sa-update
+	kill -SIGHUP $_pid
+fi
```
Comment (tcely) [inline main/spamassassin/spamd.crond]:
```suggestion

```

File: main/spamassassin/spamd.crond [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
-/usr/bin/sa-update
+set -e
+
+. /etc/conf.d/spamd
+pidfile=${pidfile:-/run/spamd.pid}
+
+if [ -f "$pidfile" ]; then
+	_pid=`cat $pidfile`
+	/usr/bin/sa-update
+	kill -SIGHUP $_pid
```
Comment (tcely) [inline main/spamassassin/spamd.crond]:
```suggestion
```

File: main/spamassassin/spamd.crond [hunk 4]
```diff
@@ -1,3 +1,13 @@
...
 
-/usr/bin/sa-update
+set -e
+
+. /etc/conf.d/spamd
+pidfile=${pidfile:-/run/spamd.pid}
+
+if [ -f "$pidfile" ]; then
+	_pid=`cat $pidfile`
+	/usr/bin/sa-update
```
Comment (tcely) [inline main/spamassassin/spamd.crond]:
```suggestion
```

File: main/spamassassin/spamd.crond [hunk 5]
```diff
@@ -1,3 +1,13 @@
 #!/bin/sh
 
-/usr/bin/sa-update
+set -e
+
+. /etc/conf.d/spamd
+pidfile=${pidfile:-/run/spamd.pid}
+
+if [ -f "$pidfile" ]; then
+	_pid=`cat $pidfile`
```
Comment (tcely) [inline main/spamassassin/spamd.crond]:
```suggestion
```

File: main/spamassassin/spamd.crond [hunk 6]
```diff
@@ -1,3 +1,13 @@
 #!/bin/sh
 
-/usr/bin/sa-update
+set -e
+
+. /etc/conf.d/spamd
+pidfile=${pidfile:-/run/spamd.pid}
+
+if [ -f "$pidfile" ]; then
```
Comment (tcely) [inline main/spamassassin/spamd.crond]:
```suggestion
```

**Comment by bratkartoffel** [comment]
Thanks @tcely for reviewing, i've changed the cron script

**Pull request closed without merging**